packages/polaris-test/tests/mandatory-headers.test.ts

```diff
@@ -0,0 +1,80 @@
+import { graphQLRequest } from '../test-utils/graphql-client';
+import { createServers } from '../test-utils/tests-servers-util';
+import * as allBooks from './jsonRequestsAndHeaders/allBooks.json';
+import * as introspectionQuery from './jsonRequestsAndHeaders/introspectionQuery.json';
+import axios from 'axios';
+import * as polarisProperties from '../shared-resources/polaris-properties.json';
+
+describe('mandatory headers', () => {
+    test.each(createServers({ allowMandatoryHeaders: true }))(
```

Comment:
define this config as a const above this test suite, and pass it every time you create servers for a test